### PR TITLE
refactor(`README`): updated component status

### PR DESCRIPTION
**Pull request by mfranzke**
Several components have been implemented, but their status hasn't been reflected within the `README`.

Updated status for the following components:
- Alert
- Drawer
- Infotext
- Link
- Section

Added:
- Brand
- NavigationItem

### CODE REVIEW

**Comment by mfranzke** [comment]
@nmerget let's align again about the status within the `complete` column, as either more or less of these and the other components might need an update.

**Comment by nmerget** [comment]
Maybe we can simplify the list, just combining the checkmarks and adding a column with the state `alpha`/`beta`/`release`?

**Comment by mfranzke** [comment]
> Maybe we can simplify the list, just combining the checkmarks and adding a column with the state `alpha`/`beta`/`release`?

I've updated the list.

File: README.md [hunk 1]
```diff
@@ -52,65 +51,67 @@ Here is an overview of all components you can use:
...
+| Floating-Button                                                     | 02 - Action       |   ❌   |        ❌        |
+| Footer                                                              | 01 - Layout       |   ❌   |        ✅        |
+| Grid                                                                | 01 - Layout       |   ❌   |        ✅        |
+| Header                                                              | 01 - Layout       |   ❌   |        ✅        |
+| [Icons](packages/components/src/components/icon)                    | 04 - Data-Display |  Beta  |        ✅        |
+| Image                                                               | 04 - Data-Display |   ❌   |        ✅        |
+| Indicator                                                           | 04 - Data-Display |   ❌   |        ❌        |
+| [Infotext](packages/components/src/components/infotext)             | 04 - Data-Display |  Beta  |        ✅        |
+| [Input](packages/components/src/components/input)                   | 03 - Data-Input   | Alpha  |        ✅        |
+| [Link](packages/components/src/components/link)                     | 02 - Action       |   ❌   |        ✅        |
```
Comment (nmerget) [inline README.md:84]: Should be `Alpha` as well

Comment (mfranzke) [reply README.md:84]: okay

**Review by nmerget — APPROVED** [review]

**Pull request merged**